<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #252525;
color:#f9f9f9;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelCurrentRoundNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Текущий номер раунда:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAboutControlTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Для смены текущего раунда щелкните правой кнопкой мыши по нужной строке.
Чтобы удалить раунд, щелкните правой кнопкой мыши по нужной строке</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddRound">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#469EC2;
border-radius:4;
color:#252525;</string>
    </property>
    <property name="text">
     <string>Добавить раунд</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartRound">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:green;
border-radius:4;
color:#252525;</string>
    </property>
    <property name="text">
     <string>Начать раунд</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStopRound">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red;
border-radius:4;
color:#f9f9f9;</string>
    </property>
    <property name="text">
     <string>Остановить раунд</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetMain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>1011</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color:#252525;
color:#252525;
background-color:#f9f9f9;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatusData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Получаемый сигнал:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Статус соединения</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPorts">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>70</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="viewPortListeningStatus">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>40</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelPortListeningStatus">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#252525;</string>
     </property>
     <property name="text">
      <string>listeting stoped</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnStartPortListening">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>100</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Medium</family>
      <pointsize>8</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:green;
border-radius:4;
color:#252525;</string>
    </property>
    <property name="text">
     <string>Начать слушать соединение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStopPortListening">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>130</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Medium</family>
      <pointsize>8</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red;
border-radius:4;
color:#f9f9f9;</string>
    </property>
    <property name="text">
     <string>Отключить прослушивание</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Статус раунда</string>
    </property>
   </widget>
   <widget class="QFrame" name="viewRoundStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelRoundStatus">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#252525;</string>
     </property>
     <property name="text">
      <string>- stopped -</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="inputJudge1Name">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>530</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#f9f9f9;
color:#000;
padding:4px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>532</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1. Судья</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnJudge1Update">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>530</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:green;
border-radius:4;
color:#252525;</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>562</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2. Судья</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputJudge2Name">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>560</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#f9f9f9;
color:#000;
padding:4px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnJudge2Update">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>560</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:green;
border-radius:4;
color:#252525;</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>592</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3. Судья</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputJudge3Name">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>590</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#f9f9f9;
color:#000;
padding:4px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnJudge3Update">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>590</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:green;
border-radius:4;
color:#252525;</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
